tutorials/issue1235.ipynb

```diff
@@ -0,0 +1,494 @@
+{
```

Comment:
<p>We need an explanation of the concepts of intensive and extensive quantities. Also, I think the list should be moved later, e.g. to the subsections "Projection of intensive/extensive quantities".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>